site/gatsby-site/netlify/functions/prismicWebhook.ts

```diff
@@ -0,0 +1,76 @@
+const axios = require("axios");
+
+exports.handler = async (event: any) => {
+  if (event.httpMethod !== "POST") {
+    return {
+      statusCode: 405,
+      body: JSON.stringify({ error: "Method not allowed" }),
+    };
+  }
+
+  try {
+    const PRISMIC_SECRET = process.env.PRISMIC_SECRET;
```

Comment:
Unless a very specific reason we should use the config object, like is this necessary for the app work? We should make sure it is defined otherwise crash the app.